Civi/Api4/Generic/BasicReplaceAction.php

```diff
@@ -95,7 +95,7 @@ public function _run(Result $result) {
     }
 
     $idField = $this->getSelect()[0];
-    $toDelete = array_diff_key(array_column($items, NULL, $idField), array_flip(array_filter(\CRM_Utils_Array::collect($idField, $this->records))));
```

Comment:
Is dropping array_filter OK here?

Reply:
Yes because it ultimately gets passed into `array_diff_key` so we only care about the difference.